gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/io/gryo/GryoVersion.java

```diff
@@ -305,6 +307,7 @@ public String getVersion() {
             add(GryoTypeReg.of(RangeGlobalStep.RangeBiOperator.class, 114));
             add(GryoTypeReg.of(OrderGlobalStep.OrderBiOperator.class, 118, new JavaSerializer()));
             add(GryoTypeReg.of(ProfileStep.ProfileBiOperator.class, 119));
+            add(GryoTypeReg.of(VertexTraverserSet.class, 173));
```

Comment:
please move the "last id" comment down from:

https://github.com/apache/tinkerpop/pull/778/files#diff-f1eb3822f6d5659dd2bdc913ae475241R208

you might need to create a separate PR against master for us to review because of this change because Gryo 3.0 will need a similar entry on master (i suspect the kryo id will be different) and we'd need to be sure this all works on master as well.

Reply:
master PR: https://github.com/apache/tinkerpop/pull/781
I have make both tp32 and tp33 ids the same. so I skipped 171,172 in  tp32

Reply:
Ok - we'd not skipped before, but it doesn't hurt anything. thanks for the second PR. building it now with docker